src/pages/SubmissionHistory/SubmissionHistory.js

```diff
@@ -134,35 +133,32 @@ export const SubmissionHistory = () => {
                 paddingTop: '1em'
               }),
               columns: [
-                ...getConfig().isActionMenuEnabled ?
```

Comment:
If a submission is finished, do we still want an empty action menu next to all the workflows?

Reply:
Are you suggesting that the action column disappear for a finished submission? Currently, the only action is `Abort` and is disabled for terminal submissions. Do we think there are other Actions that may be part of this column in the future?